docs/modules/user-guide/partials/proc_devfile-properties.adoc

```diff
@@ -0,0 +1,258 @@
+[id="proc_devfile-properties_{context}"]
+= Devfile properties
+
+[role="_abstract"]
+This page quickly introduces you to devfile properties. The code snippets help you get started developing your own devfile projects. The tables show you what the different devfile properties require if you use these properties. See the following table of contents to go to the devfile property you want to work with:
+
```

Comment:
I wonder if there is a way to make it more concise. Comparing to the original version from the odo doc, that one looks much more like a quick reference guide than this. In order to find the information that I need, I'll need to scroll through the elements here. @kadel what do you think?

Reply:
Yeh, in the current format it harder to read :-/ 
The big advantage of how it is displayed in odo.dev is that the example is on the definitions and examples are side-by-side.

Generating html version of this in a similar format like we have in https://odo.dev/file-reference/#devfile-properties would help with readability a lot. 

Just for reference markdown source for that page is at https://github.com/openshift/odo/blob/gh-pages/slate/source/index.html.md

Reply:
Thanks Elson and Tomas. Not sure if you all are able to see the rendered output, so I included some screenshots. Yes, this doc I wrote doesn't have the 3 sections like the odo table...I'm not sure if we're even able to do that in Asciidoc. I've reached out to more writers on my team and will do more research on converting Markdown to Asciidoc. I'll keep everyone posted on how much Markdown capabilities we can replicate and utilize in Asciidoc. 
<img width="1334" alt="SS of rendered odo table replica " src="https://user-images.githubusercontent.com/70717303/120698684-60dd3d00-c47d-11eb-9b86-9241a00d69f3.png">
<img width="1357" alt="2nd SS of rendered odo table replica" src="https://user-images.githubusercontent.com/70717303/120698690-62a70080-c47d-11eb-9651-5f917a048b0d.png">

Reply:
@elsony and @kadel I was talking to the other writers about Asciidoc and Antora capabilities. Using Antora to build our docs, we don't have the ability to replicate the three pane view that's used in the odo page. I was also recommended to use lists instead of tables....tables are harder to edit, build, and use for accessibility purposes. I've included some screenshots of the rendered version. As you can see, the horizontal lists mirrors the table pretty well. 

Let me know what you guys think, thanks! 
<img width="1183" alt="Screen Shot 2021-06-07 at 3 22 17 PM" src="https://user-images.githubusercontent.com/70717303/121076595-972ffa80-c7a4-11eb-9db5-aabb9391aca1.png">
<img width="1178" alt="Screen Shot 2021-06-07 at 3 22 27 PM" src="https://user-images.githubusercontent.com/70717303/121076613-9a2aeb00-c7a4-11eb-889d-0c8fe8c50ac0.png">
<img width="1214" alt="Screen Shot 2021-06-07 at 3 22 34 PM" src="https://user-images.githubusercontent.com/70717303/121076623-9c8d4500-c7a4-11eb-8ad3-81ce60ec79c9.png">

Reply:
I'm sorry to jump in @elsony and @kadel, writers aren't big web design specialists, ofc, but there're some related topics that we discuss and research, e.g. user experience and accessibility.

It can be just me, but I don't see how this wide and packed 3-pane layout https://odo.dev/file-reference/#devfile-properties is easy to use, sorry.

For me the 2nd and 3rd panes sometimes get an extra scroll bar at the cell bottom, information gets hidden, it means extra clicking and scrolling, more time to get info, not user-friendly at all.
If we zoom out to see all content, it'll be very wide, meaning more eye movement and more time spent on a page again. IIRC, UI/UX designers' work is to provide info faster and make users click less.

Accessibility-wise, how can we be sure that all screen readers correctly connect code examples with devfile properties? Tables are a big accessibility concern on their own, and these 3 panes with tables are potentially very confusing to screen readers.

In short, @jc-berger 's proposed design might make a page longer, but in the end it provides information faster and in a more straightforward and accessible way. Red Hat docs writers are currently trained to choose simpler and more lightweight structures, hence Jake's proposal, I guess.

Reply:
> It can be just me, but I don't see how this wide and packed 3-pane layout https://odo.dev/file-reference/#devfile-properties is easy to use, sorry.

The feedback that we got on this layout was only positive.